cfg/inputinclude.go

```diff
@@ -49,6 +44,7 @@ func (in *InputInclude) clone() *InputInclude {
 	var clone InputInclude
 
 	deepcopy.MustCopy(in, &clone)
+	// TODO why is filepath assignment manually and not cloned?
```

Comment:
Strings are immutable, so there is no need to clone.

Reply:
thanks :-)